components/Starknet/modules/architecture_and_concepts/pages/Accounts/approach.adoc

```diff
@@ -3,10 +3,10 @@
 
 A Starknet account contract must include the following two functions:
 
-* `&lowbar;&lowbar;validate&lowbar;&lowbar;`
-* `&lowbar;&lowbar;execute&lowbar;&lowbar;`
+* `+__validate__+`
+* `+__execute__+`
 
-These serve distinct purposes to ensure that only the account owner can initiate transactions and that fees can be charged for the resources you use.
+These functions serve distinct purposes to ensure that only the account owner can initiate transactions and that fees can be charged for the resources they use.
 
 Starknet's account type is inspired by Ethereum's EIP-4337, where instead of EOAs, you now use smart contract accounts with arbitrary verification logic.
```

Comment:
```suggestion
Starknet's account type is inspired by Ethereum's EIP-4337, where instead of an account being an EOA, it is now a smart contract account with arbitrary verification logic.
```